### ISSUE TITLE
Google Analytics URLs

### ISSUE THREAD

**Issue by cristiandelgadod** [w3c/webappsec-csp#648]
Currently, we have set up the CSP header like this:
`
Content-Security-Policy:
default-src 'self' 'unsafe-inline' 'unsafe-eval' 'self' https://*.sitecorecloud.io https://*.moosend.com https://*.bootstrapcdn.com https://*.google.com https://*.gstatic.com https://darling-development.vercel.app https://darling-qa.vercel.app https://dii-qa-wip.vercel.app https://darling-demo.vercel.app https://dii-demo-wip.vercel.app https://dii-stage.vercel.app https://dii-stage-wip.vercel.app https://dii-prod.vercel.app https://dii-prod-wip.vercel.app https://*.darlingii.com https://*.m-operations.com https://*.pusher.com wss://*.pusher.com https://vercel.live https://rebrand-mc5viovu7-darlingii.vercel.app https://cdn.cookielaw.org https://*.onetrust.com https://*.googletagmanager.com https://*.google-analytics.com https://*.googleadservices.com https://cdn.stat-track.com/statics/moosend-tracking.min.js https://polyfill.io/v3/polyfill.min.js https://*.stat-track.com https://forms.m-pages.com https://*.doubleclick.net https://cdn.cookielaw.org https://darl-p-001.sitecorecontenthub.cloud https://fonts.googleapis.com https://*.fontawesome.com; object-src 'none'; frame-ancestors 'none'; block-all-mixed-content; upgrade-insecure-requests;
`

But we got this error from our users in Vietnam:

`Refused to load the image 'https://www.google.com.vn/ads/ga-audiences?v=1&t=sr&slf_rd=1&_r=4&tid=G-VMLNB0EWQG&cid=318452825.1709107785&gtm=45je42q1v9174953861z89174950045za200&aip=1&dma=0&gcd=13l3l3l3l1&npa=0&z=1324115001' because it violates the following Content Security Policy directive:`

Given this, we would have to set up all the google.com domains for every country in the world so that GA4 tracking does not fail.

Is there an alternative to set up CSP so it allows the domains with a wildcard, like google.com.*/ads/....

**Comment by gapple** [comment]
tl;dr - no, it's not possible to use a wildcard for TLD

Previous discussion: [#256]

From Google's documentation, you can find where they publish [their full list of domains](https://www.google.com/supported_domains): 
https://developers.google.com/tag-platform/security/guides/csp#google_analytics_4_google_analytics
